### PR TITLE
⬆️ Upgraded scDblFinder Image

### PR DESCRIPTION
**Pull request by migbro**
## Description

Continuing the work of improving the Single Cell docker images. Build time is not much faster, but nearly 2GB smaller

Fixes # https://github.com/d3b-center/bixu-tracker/issues/2186

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [Test A](https://cavatica.sbgenomics.com/u/d3b-bixu/kf-dev-single-cell-rna/tasks/0c0fb64d-3009-44f3-998a-cc9ef41a873e/)
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

**Review by dmiller15 — APPROVED** [review]

File: scdblfinder/v1.12.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM pgc-images.sbgenomics.com/d3b-bixu/scrna_qc:v1.0.1
+
+LABEL maintainer="Miguel Brown (brownm28@chop.edu)"
+LABEL description="scDblFinder - use qc docker as base image as usage in workflow is wrapped in an R markdown notebook. Heavy stuff"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt install -y libbz2-dev libxt-dev
```
Comment (sakshamphul) [inline scdblfinder/v1.12.0/Dockerfile:8]: is update enough without upgrade?

Came across this at https://askubuntu.com/questions/94102/what-is-the-difference-between-apt-get-update-and-upgrade ?

> apt-get update updates the list of available packages and their versions, but it does not install or upgrade any packages.
apt-get upgrade actually installs newer versions of the packages you have. After updating the lists, the package manager knows about available updates for the software you have installed. This is why you first want to update.

**Review by sakshamphul — COMMENTED** [review]
Dropped a comment on update vs upgrade

File: scdblfinder/v1.12.0/Dockerfile (hunk 1 above)
Comment (migbro) [reply scdblfinder/v1.12.0/Dockerfile:8]: Um, probably can't hurt. Update for sure is needed as over time the list of available packages gets stale. The autoremove tip at the end might be useful to clean up in the end. I can add it on and see

Comment (migbro) [reply scdblfinder/v1.12.0/Dockerfile]: Ok, added. Seems fine

**Review by sakshamphul — APPROVED** [review]

**Pull request merged**